docs/hbar_transfer.md

```diff
@@ -0,0 +1,38 @@
+# Transferring Hbar Using the Hedera JSON-RPC Relay
+
+## Add Hedera Network to Metamask
```

Comment:
Please add a sentence above noting that any JSON RPC compatible wallet is applicable as they just need to be able to call our endpoints.
Then say for illustration purposes only we are using Metamask as an example

Reply:
done